definitions/consented/json/ComplexTypes.json

```diff
@@ -665,6 +665,7 @@
     "FieldType": "FixedList",
     "FieldTypeParameter": "FR_Gender",
     "ElementLabel": "Gender",
+    "FieldShowCondition": "gender=\"Male\" OR gender=\"Female\" OR gender=\"male\" OR gender=\"female\" ",
```

Comment:
Adding hide if its not set to Male,Female or it's lower case values (was lower case with legacy data).
Doesn't impact entry as the data comes from BSP. Means "notGiven" is not shown.

Reply:
why do we need to hide it is its not one of these?

Reply:
If they don't supply it then it gets mapped as "notGiven" and takes up space on the screen. This is just to hide it if its 'notGiven'